realm/realm-jni/src/io_realm_internal_table.cpp

```diff
@@ -1384,16 +1400,16 @@ JNIEXPORT jlong JNICALL Java_io_realm_internal_Table_nativeGetSortedView(
     switch (colType) {
         case type_Int:
         case type_Bool:
-        case type_DateTime:
         case type_String:
         case type_Double:
         case type_Float:
+        case type_Timestamp:
             try {
                 TableView* pTableView = new TableView( pTable->get_sorted_view(S(columnIndex), ascending != 0 ? true : false) );
                 return reinterpret_cast<jlong>(pTableView);
             } CATCH_STD()
         default:
-            ThrowException(env, IllegalArgument, "Sort is currently only supported on integer, boolean, double, float, String, and Date columns.");
+            ThrowException(env, IllegalArgument, "Invalid type - Only String, Date, boolean, byte, short, int, long and their boxed variants are supported.");
```

Comment:
Was this deliberately re-worded to be the same as the `FindAllString` exception above or is it a copy-and-paste error? It loses the original's mention of **Sort**.